packages/core/src/interfaces/Icon.ts

```diff
@@ -22,7 +20,12 @@ export interface IconSlot {
   Icon: IconI;
 }
 
+type $Without<T, K> = Pick<T, Exclude<keyof T, K>>;
```

Comment:
> Isn't this similar to `Omit`?

It is, does the same thing basically. I did try using that first, but all the types for the props were lost for some reason. I'm thinking it's conflicting with the ThemeProvider in how it also excludes the theme prop. It would be probably be best to update our outdated [theme provider package](https://www.npmjs.com/package/@draftbit/react-theme-provider). But this will do for now.